src/entity.js

```diff
@@ -4,7 +4,7 @@ var util            = require('util');
 var slugid          = require('slugid');
 var _               = require('lodash');
 var Promise         = require('promise');
-var debug           = require('debug')('base:entity');
+var debug           = require('debug')('entity');
```

Comment:
As much as I love this change, I think that we probably have some `DEBUG=` patterns which use the old `base:` prefix to filter entity logging.  I wonder if the pragmatic approach here is to leave the debug name and break things completely, but only once, when we move to structured logging?  I don't have strong feelings on this.

Reply:
Yeah, good point. I'll switch it back!